components/MultiTokenActivityList/index.tsx

```diff
@@ -124,24 +124,24 @@ const ActivityList: React.FC<
         <tbody>
           {transfers?.metadata.total_count ? (
             transfers.entries.map(item => {
-              const method = item.transaction.method_name || item.transaction.method_id
+              const method = item.transaction?.method_name || item.transaction?.method_id
```

Comment:
The entire record should be removed if `item.transaction` is `undefined`. Besides, we should find out why `item.transaction` is `undefined`, it should be a valid value

Reply:
yep. It seems a problem in the response.  And to avoid the same crash, I added this treatment.
<img width="1136" alt="image" src="https://user-images.githubusercontent.com/22511289/230812276-fba1d207-7440-4f68-852e-c007f820e6a8.png">

Could you please help to find the reason? @Naupio